plugins/cgroups/util-cgroups-plugins/tests/tests.rs

```diff
@@ -0,0 +1,480 @@
+use std::{
+    path::{Path, PathBuf},
+    time::{Duration, UNIX_EPOCH},
+};
+
+use alumet::{
+    agent::{
+        self,
+        plugin::{PluginInfo, PluginSet},
+    },
+    measurement::{
+        AttributeValue, MeasurementAccumulator, MeasurementBuffer, MeasurementPoint, Timestamp, WrappedMeasurementValue,
+    },
+    metrics::TypedMetricId,
+    pipeline::{
+        Source,
+        elements::{error::PollError, source::trigger::TriggerSpec},
+        naming::TransformName,
+    },
+    plugin::{PluginMetadata, rust::AlumetPlugin},
+    resources::{Resource, ResourceConsumer},
+    test::{
+        RuntimeExpectations,
+        runtime::{TransformCheckInputContext, TransformCheckOutputContext},
+    },
+    units::Unit,
+};
+use anyhow::Context;
+use util_cgroups::{CgroupHierarchy, CgroupVersion, hierarchy::find_user_app_slice};
+
+use serde::{Deserialize, Serialize};
+use util_cgroups_plugins::{
+    cgroup_events::CgroupFsMountCallback,
+    job_annotation_transform::{
+        CachedCgroupHierarchy, JobAnnotationTransform, JobTagger, OptionalSharedHierarchy, SharedCgroupHierarchy,
+    },
+};
+
+use lazy_static::lazy_static;
+
+const SYSFS_CGROUP: &str = "/sys/fs/cgroup";
+
+lazy_static! {
```

Comment:
these are used in only one function, you can declare them locally (in the function) and remove the dependency